Orange/widgets/visualize/owradviz.py

```diff
@@ -353,6 +353,10 @@ def primitive_variables(self):
     def effective_variables(self):
         return self.selected_vars
 
+    @property
+    def effective_data(self):
+        return self.data.transform(Domain(self.effective_variables))
+
```

Comment:
I'm not qualified to check this, I either never knew or I forgot what `effective_data` does. @VesnaT, can you please approve this?